pom.xml

```diff
@@ -1276,6 +1279,19 @@
               <include>org.spark-project.spark:unused</include>
             </includes>
           </artifactSet>
+          <relocations>
+            <relocation>
+              <pattern>com.google.common</pattern>
+              <shadedPattern>org.spark-project.guava</shadedPattern>
+              <excludes>
+                <exclude>com/google/common/base/Absent*</exclude>
```

Comment:
mind adding a quick note here to explain that we expose these in our API so we don't relocate them?